src/types/track_properties.rs

```diff
@@ -0,0 +1,12 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use super::AudioInputId;
+
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+pub struct TrackProperties {
+    input_id: AudioInputId,
+    // Temporal
+    volume: Option<u32>,
+    // ...
+}
```

Comment:
In the first version, I don't want to use any params. All audio tracks would be mixed in the simplest possible way